### PR TITLE
Update documentation to include Protocol 19 features

### PR DESCRIPTION
**Pull request by Shaptic**
**Note:** This shouldn't be merged until at least the testnet is upgraded.

This closes the required items in #668.

### CODE REVIEW

File: content/api/resources/transactions/object.mdx [hunk 1]
```diff
@@ -69,10 +69,40 @@ When Horizon returns information about a transaction, it uses the following form
...
+- preconditions
+  - object
+  - A set of transaction preconditions determining its validity.
+    - time_bounds
+      - min_time
+        - string, encoding a 64-bit UNIX timestamp
+        - The timestamp after which a transaction is valid.
+      - max_time
+        - string, encoding a 64-bit UNIX timestamp
+        - The timestamp before which a transaction is valid.
```
Comment (leighmcculloch) [inline content/api/resources/transactions/object.mdx]: Suggest including signedness that will affect how people parse the string.
```suggestion
      - min_time
        - string, containing an unsigned 64-bit integer UNIX timestamp
        - The timestamp after which a transaction is valid.
      - max_time
        - string, containing an unsigned 64-bit integer UNIX timestamp
        - The timestamp before which a transaction is valid.
```

File: content/api/resources/transactions/object.mdx [hunk 2]
```diff
@@ -69,10 +69,40 @@ When Horizon returns information about a transaction, it uses the following form
...
+    - ledger_bounds
+      - min_ledger
+        - number
+        - The lowest ledger number for which this transaction is valid.
+      - max_ledger
+        - number
+        - The highest ledger number for which this transaction is valid.
```
Comment (leighmcculloch) [inline content/api/resources/transactions/object.mdx]: Suggest including what the valid range is for these values.
```suggestion
    - ledger_bounds
      - min_ledger
        - number, containing an unsigned 32-bit integer
        - The lowest ledger number for which this transaction is valid.
      - max_ledger
        - number, containing an unsigned 32-bit integer
        - The highest ledger number for which this transaction is valid.
```

File: content/api/resources/transactions/object.mdx [hunk 3]
```diff
@@ -69,10 +69,40 @@ When Horizon returns information about a transaction, it uses the following form
...
+    - min_account_sequence
+      - string, encoding a 64-bit sequence number
```
Comment (leighmcculloch) [inline content/api/resources/transactions/object.mdx]:
```suggestion
      - string, containing a positive-only signed 64-bit integer
```

File: content/api/resources/transactions/object.mdx [hunk 4]
```diff
@@ -69,10 +69,40 @@ When Horizon returns information about a transaction, it uses the following form
...
+      - The lowest source account sequence number for which the transaction is valid.
+    - min_account_sequence_age
+      - number, representing a duration in seconds
```
Comment (leighmcculloch) [inline content/api/resources/transactions/object.mdx]:
```suggestion
      - number, containing an unsigned 64-bit integer duration in seconds
```

File: content/api/resources/transactions/object.mdx [hunk 5]
```diff
@@ -69,10 +69,40 @@ When Horizon returns information about a transaction, it uses the following form
...
+      - The minimum duration of time that must have passed since the source account's sequence number changed for the transaction to be valid.
+      - The minimum amount of time from the last time an account's sequence number changed that must have passed for the transaction to be valid.
+    - min_account_sequence_ledger_gap
+      - number
```
Comment (leighmcculloch) [inline content/api/resources/transactions/object.mdx:104]:
```suggestion
      - number, containing an unsigned 32-bit integer
```

File: content/api/resources/transactions/object.mdx [hunk 6]
```diff
@@ -69,10 +69,40 @@ When Horizon returns information about a transaction, it uses the following form
...
+      - The minimum number of ledgers that must have closed since the source account's sequence number changed for the transaction to be valid.
+    - extra_signers
+      - array of strings
```
Comment (leighmcculloch) [inline content/api/resources/transactions/object.mdx:107]: Suggest including the size constraint.
```suggestion
      - array of strings, max count 2
```

File: content/docs/glossary/multisig.mdx [hunk 7]
```diff
@@ -55,6 +55,10 @@ If the weight of the master key is updated to 0, it cannot be used to sign trans
 
 Each additional signer beyond the master key increases the account's [minimum balance](./minimum-balance.mdx).
 
+## One-Time Extra Signers
+
+You can require that a transaction have a particular signer that isn't associated with any of the accounts used in the transaction by including it in the [transaction validity precondition](./transactions.mdx#extra-signers).
```
Comment (leighmcculloch) [inline content/docs/glossary/multisig.mdx:60]: Suggest using ubiquitous language for the title, and noting the one-time quality in the body.
```suggestion
## Extra Signers

You can require that a transaction have a particular signer that isn't associated with any of the accounts used in the transaction by including it in the [transaction validity precondition](./transactions.mdx#extra-signers). The extra signer applies only to that one transaction.
```

File: content/docs/glossary/multisig.mdx [hunk 8]
```diff
@@ -69,10 +73,18 @@ This type of signer is especially useful in escrow accounts. You can pre-authori
...
+Adding a signer of type hash(x) allows anyone who knows `x` to sign the transaction. This type of signer is especially useful in [atomic cross-chain swaps](https://en.bitcoin.it/wiki/Atomic_cross-chain_trading) which are needed for inter-blockchain protocols like [lightning networks](https://lightning.network).
 
 First, create a random 256 bit value, which we call `x`. The SHA256 hash of that value can be added as a signer of type hash(x). Then in order to authorize a transaction, `x` is added as one of the signatures of the transaction. Keep in mind that `x` will be known to the world as soon as a transaction is submitted to the network with `x` as a signature. This means anyone will be able to sign for that account with the hash(x) signer at that point. Often you want there to be additional signers so someone must have a particular secret key and know `x` in order to reach the weight threshold required to authorize transactions on the account.
 
+### Signed Payloads
+
+Adding a signed payload as a signer means that transactions with a signature on that payload by that signer are valid.
```
Comment (leighmcculloch) [inline content/docs/glossary/multisig.mdx:82]: Suggest some changes for subtle clarity:
```suggestion
Adding a signed payload as a signer means that transactions with a signature, of a specified payload by a specified key, are valid.
```

File: content/docs/glossary/multisig.mdx [hunk 9]
```diff
@@ -69,10 +73,18 @@ This type of signer is especially useful in escrow accounts. You can pre-authori
...
+
+More verbosely, you add a signer of `(public key, payload)`, where the public key is the Stellar address of the expected signer and the payload is any arbitrary data of up to 64 bytes. A valid signature, then, is a signature on that payload by the private key corresponding to the public key. This is also described in [CAP-40](https://stellar.org/protocol/cap-40) which introduced this new signer.
```
Comment (leighmcculloch) [inline content/docs/glossary/multisig.mdx:84]:
```suggestion
More verbosely, you add a signer of `(public key, payload)`, where the public key is a Stellar public key of the expected signer and the payload is any arbitrary data of 1 to 64 bytes in length. A valid signature, then, is a signature of that payload by the private key corresponding to the public key. This is also described in [CAP-40](https://stellar.org/protocol/cap-40) which introduced this new signer.
```

File: content/docs/glossary/multisig.mdx [hunk 10]
```diff
@@ -69,10 +73,18 @@ This type of signer is especially useful in escrow accounts. You can pre-authori
...
+
+Like with [hash signers](#hashx), once the signature is revealed to the world, anyone would be able to use the signature as a valid signer.
```
Comment (leighmcculloch) [inline content/docs/glossary/multisig.mdx:86]:
```suggestion
Like with [hash signers](#hashx), once signed payload signature is revealed to the world, anyone is able to use the signature to authorize transactions that are contingent on the signed payload signer.
```

File: content/docs/glossary/transactions.mdx [hunk 11]
```diff
@@ -42,10 +42,39 @@ The memo contains optional extra information. It is the responsibility of the cl
...
+
+#### Time Bounds
 The optional UNIX timestamp (in seconds), determined by ledger time, of a lower and upper bound of when this transaction will be valid. If a transaction is submitted too early or too late, it will fail to make it into the transaction set. `maxTime` equal `0` means that it's not set. _We highly advise for all transactions to use time bounds, and many SDKs enforce their usage._ If a transaction doesn't make it into the transaction set, it is kept around in memory in order to be added to the next transaction set on a best-effort basis. Because of this behavior, we highly advise that all transactions are created with time bounds in order to invalidate transactions after a certain amount of time, especially if you plan to resubmit your transaction at a later time.
 
+#### Ledger Bounds
+These are like [Time Bounds](#time-bounds), except they apply to ledger numbers. With them set, a transaction will only be valid for ledger numbers that fall into the range you set. The lower bound is inclusive while the upper bound is not. If you set the upper bound to zero, this indicates that there is no upper bound.
```
Comment (leighmcculloch) [inline content/docs/glossary/transactions.mdx:55]: >The lower bound is inclusive while the upper bound is not.

This is incorrect. The upper bound is inclusive.

See https://github.com/stellar/stellar-core/blob/da846c923bbd989fe716c9a3eb32e04ef2ae37db/src/transactions/TransactionFrame.cpp#L491-L496.

cc @sisuresh for confirmation

File: content/docs/glossary/transactions.mdx [hunk 12]
```diff
@@ -42,10 +42,39 @@ The memo contains optional extra information. It is the responsibility of the cl
...
+
+#### Minimum Sequence Number
+If this precondition is set, the transaction will only be valid when its source account's sequence number (call it `S`) is large enough. Specifically, it's valid when `S` satisfies `minSeqNum <= S < tx.seqNum`. If this precondition is omitted, the default behavior applies: the transaction's sequence number must be exactly one greater than the account's sequence number.
+
+Note that after a transaction is executed, the account will always raise its sequence number to the transaction's sequence number.
```
Comment (leighmcculloch) [inline content/docs/glossary/transactions.mdx:60]: I think this is subtle, but I think technically we describe this as setting, not raising, because it doesn't work like bumpseq's where it only changes it if it is lower. This is not particularly meaningful because of the constraint described in the prior paragraph we know it nevers goes backwards, but mostly we describe it as 'set' in our own conversations.
```suggestion
Note that after a transaction is executed, the account will always set its sequence number to the transaction's sequence number.
```

File: content/docs/glossary/transactions.mdx [hunk 13]
```diff
@@ -42,10 +42,39 @@ The memo contains optional extra information. It is the responsibility of the cl
...
+
+#### Minimum Sequence Age
+With this precondition, the transaction is only valid after an account's last sequence number change exceeds a particular duration (expressed in seconds).
```
Comment (leighmcculloch) [inline content/docs/glossary/transactions.mdx:63]: This is subtly incorrect.
```suggestion
Minimum sequence age makes a transaction valid only if a duration of time (specified in seconds) has passed since the last time that the sequence number for a source account has been operated on, either by a BumpSequence operation, or a Transaction. Note that a BumpSequence resets this period regardless of whether the sequence number of the source account is changed.
```

File: content/docs/glossary/transactions.mdx [hunk 14]
```diff
@@ -42,10 +42,39 @@ The memo contains optional extra information. It is the responsibility of the cl
...
+
+With a minimum [time bound](#time-bounds), the transaction becomes valid after a particular amount of time passes unconditionally. With this precondition, the passage of time applies only to the transaction's source account: you can allow validity after an *account* has waited long enough since its last sequence number change.
```
Comment (leighmcculloch) [inline content/docs/glossary/transactions.mdx:65]: This paragraph is confusing, and it feels unrelated to this section, or out of place. Is it a copy-paste mistake?

File: content/docs/glossary/transactions.mdx [hunk 15]
```diff
@@ -42,10 +42,39 @@ The memo contains optional extra information. It is the responsibility of the cl
...
+
+#### Minimum Sequence Ledger Gap
+With this precondition, the transaction is only valid after the current network ledger number meets (or exceeds) a particular gap relative to the ledger corresponding to the source account's last sequence number change.
```
Comment (leighmcculloch) [inline content/docs/glossary/transactions.mdx:68]: Same as above:
```suggestion
Minimum sequence ledger gap makes a transaction valid only if a duration of ledgers has passed since the last ledger that the sequence number for a source account has been operated on, either by a BumpSequence operation, or a Transaction. Note that a BumpSequence resets this period regardless of whether the sequence number of the source account is changed.
```

File: content/docs/glossary/transactions.mdx [hunk 16]
```diff
@@ -42,10 +42,39 @@ The memo contains optional extra information. It is the responsibility of the cl
...
+
+This is similar to the [minimum sequence age](#minimum-sequence-age), except it's expressed as a quanity of ledgers rather than a duration of time.
+
+#### Extra Signers
+You can require that the transaction have up to two additional signatures beyond the ones required for transaction's authorization in general. There must additional, one-off signatures corresponding to every extra signer in this list, even if the signature is not otherwise required by the sourceAccount or operations.
```
Comment (leighmcculloch) [inline content/docs/glossary/transactions.mdx:73]: Subtle changes regarding signers vs signatures:
```suggestion
You can require that the transaction have up to two additional signers beyond the ones required for transaction's authorization in general. For each extra signer there must be a signature corresponding to the signer, even if the signature would not otherwise be required by the transaction's source account or operations.
```

File: content/docs/glossary/transactions.mdx [hunk 17]
```diff
@@ -67,7 +96,14 @@ To determine if a transaction is valid, many checks take place over the course o
   - The appropriate network passphrase was part of the transaction hash that was signed by each of the signers. See [Network Passphrases](./network-passphrase.mdx) for more.
   - The combined weight of all signatures for the source account _of the transaction_ meets the low threshold for the source account. This is necessary in order for fees to be taken and the sequence number to be incremented later in the transaction lifecycle.
 - **Memo** — The memo type must be a valid type, and the memo itself must be adhere to the formatting of the memo type.
-- **Time Bounds** — The transaction must be submitted within the set time bounds of the transaction, otherwise it will be considered invalid.
+- **Preconditions** — refer to the [Validity Conditions](#validity-conditions) described above for thorough explanations of each of these. For each of these that are set, the transaction will be considered invalid unless the transaction is submitted:
+  * **Time Bounds** — within the set time bounds of the transaction
+  * **Ledger Bounds** — within the set ledger bounds of the transaction
+  * **Minimum Sequence Number** — by a source account whose sequence number is larger than this value
```
Comment (leighmcculloch) [inline content/docs/glossary/transactions.mdx:102]: Correction:
```suggestion
  * **Minimum Sequence Number** — by a source account whose sequence number is larger or equal to this value
```

File: content/docs/glossary/transactions.mdx [hunk 18]
```diff
@@ -67,7 +96,14 @@ To determine if a transaction is valid, many checks take place over the course o
...
+  * **Minimum Sequence Age** — by a source account who last changed its sequence number at least "x" long ago
```
Comment (leighmcculloch) [inline content/docs/glossary/transactions.mdx:103]: This is incorrect, it doesnt' have to have changed, but it is also hard to discuss it here, so I think we should just link to teh larger section.
```suggestion
  * **Minimum Sequence Age** — see [Minimum Sequence Age](#minimum-sequence-age).
```

File: content/docs/glossary/transactions.mdx [hunk 19]
```diff
@@ -67,7 +96,14 @@ To determine if a transaction is valid, many checks take place over the course o
...
+  * **Minimum Sequence Ledger Gap** — by a source account whose sequence number was last changed in a ledger at least "x" ledgers from the current ledger
```
Comment (leighmcculloch) [inline content/docs/glossary/transactions.mdx:104]: Same as above:
```suggestion
  * **Minimum Sequence Ledger Gap** — see [Minimum Sequence Ledger Gap](#minimum-sequence-ledger-gap).
```

**Review by leighmcculloch — COMMENTED** [review]
Few suggestions and requests inline.

File: content/docs/glossary/transactions.mdx (hunk 11 above)
Comment (Shaptic) [reply content/docs/glossary/transactions.mdx:55]: Then the [CAP-21 comments](https://github.com/stellar/stellar-protocol/blob/master/core/cap-0021.md#xdr-diff) are incorrect:

```diff
+    // Transaction only valid for ledger numbers n such that
+    // minLedger <= n < maxLedger (if maxLedger == 0, then
+    // only minLedger is checked)
+    LedgerBounds *ledgerBounds;
```

File: content/docs/glossary/transactions.mdx (hunk 16 above)
Comment (Shaptic) [reply content/docs/glossary/transactions.mdx:73]: By adding extra signers, you require extra signatures on the transaction. To say you can require extra signers doesn't really make sense.

Perhaps this is a better reword?
```suggestion
A transaction can specify up to two extra signers as a precondition, meaning it must have corresponding additional signatures, even if those signatures would not otherwise be required to authorize the transaction (i.e. for its source account or operations).
```

File: content/docs/glossary/transactions.mdx (hunk 17 above)
Comment (Shaptic) [reply content/docs/glossary/transactions.mdx:102]: With that, "greater than" is more conventional than "larger"

```suggestion
  * **Minimum Sequence Number** — by a source account whose sequence number is greater than or equal to this value
```

File: content/docs/glossary/transactions.mdx (hunk 14 above)
Comment (Shaptic) [reply content/docs/glossary/transactions.mdx:65]: What's confusing about it? It's relating the concept of timebound-based validity to "account did a thing"-based validity.

File: content/docs/glossary/transactions.mdx (hunk 16 above)
Comment (leighmcculloch) [reply content/docs/glossary/transactions.mdx:73]: My edit says there has to be a signature corresponding to the signer, not that it requires extra signers. An extra signer may require the same signature as an operation already does, so strictly speaking an extra signer doesn't require an extra signature, or even any additional signatures, there just has to be corresponding signatures to signers.

Your edit is fine too, if you remove the `additional` word:
```suggestion
A transaction can specify up to two extra signers as a precondition, meaning it must have signatures that correspond to those extra signers, even if those signatures would not otherwise be required to authorize the transaction (i.e. for its source account or operations).
```

File: content/docs/glossary/transactions.mdx [hunk 20]
```diff
@@ -42,10 +42,39 @@ The memo contains optional extra information. It is the responsibility of the cl
...
 
+#### Ledger Bounds
+These are like [Time Bounds](#time-bounds), except they apply to ledger numbers. With them set, a transaction will only be valid for ledger numbers that fall into the range you set. The lower bound is inclusive while the upper bound is not. If you set the upper bound to zero, this indicates that there is no upper bound.
+
+#### Minimum Sequence Number
+If this precondition is set, the transaction will only be valid when its source account's sequence number (call it `S`) is large enough. Specifically, it's valid when `S` satisfies `minSeqNum <= S < tx.seqNum`. If this precondition is omitted, the default behavior applies: the transaction's sequence number must be exactly one greater than the account's sequence number.
```
Comment (leighmcculloch) [inline content/docs/glossary/transactions.mdx:58]: I think we should avoid the "this precondition". It's helpful to imagine that the headings are gone and the paragraphs should speak for themselves.
```suggestion
If a minimum sequence number is set, the transaction will only be valid when its source account's sequence number (call it `S`) is large enough. Specifically, it's valid when `S` satisfies `minSeqNum <= S < tx.seqNum`. If this precondition is omitted, the default behavior applies: the transaction's sequence number must be exactly one greater than the account's sequence number.
```

File: content/docs/glossary/transactions.mdx (hunk 14 above)
Comment (leighmcculloch) [reply content/docs/glossary/transactions.mdx:65]: The fact it leads with discussing time bounds. Maybe rewording it like this would help:

```suggestion
Minimum sequence age is a precondition relating to time, but unlike [time bound](#time-bounds) which is an absolute time, minimum sequence age is relative to the transaction source account's sequence age field.
```

File: content/docs/glossary/transactions.mdx (hunk 16 above)
Comment (Shaptic) [reply content/docs/glossary/transactions.mdx:73]: It does say that, though, doesn't it? :smile: 

> You can require that the transaction have up to two additional signers 

Good point regarding redundant signatures, though.

File: content/api/resources/transactions/object.mdx [hunk 21]
```diff
@@ -69,10 +69,40 @@ When Horizon returns information about a transaction, it uses the following form
   - An array of signatures used to sign this transaction.
 - valid_after
   - RFC3339 date-time string
-  - The date after which a transaction is valid.
+  - The date after which a transaction is valid. This will be deprecated in Horizon v3 in lieu of `preconditions.time_bounds.min_time`.
```
[the remark this answers is not in the document]
Comment (Shaptic) [reply content/api/resources/transactions/object.mdx:72]: Yes :facepalm: good catch

File: content/api/resources/transactions/object.mdx (hunk 1 above)
Comment (Shaptic) [reply content/api/resources/transactions/object.mdx]: Yeah, good call. Putting these into the description, though, as putting them into the type makes the rendering ugly and doesn't align with how it's done elsewhere.

Comment (Shaptic) [reply content/api/resources/transactions/object.mdx]: OpenAPI spec when? :cricket:

File: content/docs/glossary/multisig.mdx (hunk 8 above)
Comment (Shaptic) [reply content/docs/glossary/multisig.mdx:82]: This is more confusing to me. "a specified" feels generic: specified where? whereas "that payload" is a back-reference to the prior mention of payload. Maybe "**the** specified"? And "specified key" is technically wrong since it's the private key that needs to sign, but it's not the one that's specified.. :laughing: this is hard to explain precisely

Comment (leighmcculloch) [reply content/docs/glossary/multisig.mdx:82]: How about:
```suggestion
Adding a signed payload as a signer means that transactions are only valid if they are accompanied by a signature of the signer's payload. The signature must be produced using the private key corresponding to the public key specified in the signer.
```

File: content/docs/glossary/multisig.mdx (hunk 9 above)
Comment (Shaptic) [reply content/docs/glossary/multisig.mdx:84]:
```suggestion
More verbosely, you add a signer of `(public key, payload)`, where the public key is the Stellar public key of the expected signer and the payload is 1 to 64 bytes of arbitrary data. A valid signature, then, is a signature of that payload by the private key corresponding to the public key. This is also described in [CAP-40](https://stellar.org/protocol/cap-40) which introduced this new signer.
```

Saying `address` is intended to explain that it's a `G...` value, but saying "where the public key is the Stellar public key" seems redundant...

File: content/docs/glossary/transactions.mdx (hunk 14 above)
Comment (Shaptic) [reply content/docs/glossary/transactions.mdx:65]: Ooh, very nice reword.

File: content/docs/glossary/transactions.mdx (hunk 20 above)
Comment (Shaptic) [reply content/docs/glossary/transactions.mdx:60]: But then what's the point of the headings? Especially since it's the first paragraph following it. Seems redundant.

File: content/docs/glossary/transactions.mdx (hunk 18 above)
Comment (Shaptic) [reply content/docs/glossary/transactions.mdx]: Yeah seems good, amended.

File: content/docs/glossary/transactions.mdx (hunk 15 above)
Comment (Shaptic) [reply content/docs/glossary/transactions.mdx]: I consolidated this into parent paragraph - _Validity Conditions_ - and clarify/introduce/define what it means for an account sequence number to be touched (I also think a term like `refreshed` could work).

File: content/docs/glossary/transactions.mdx [hunk 22]
```diff
@@ -42,10 +42,38 @@ The memo contains optional extra information. It is the responsibility of the cl
...
+
+#### Minimum Sequence Ledger Gap
+When the ledger gap precondition is set, the transaction is only valid after the current network ledger number meets (or exceeds) a particular gap relative to the ledger corresponding to the account's [sequence number age](./accounts.mdx#sequence-time-and-ledger).
+
+This is similar to the [minimum sequence age](#minimum-sequence-age), except it's expressed as a number of ledgers rather than a duration of time.
+
+#### Extra Signers
+You can require that the transaction have up to two additional signatures beyond the ones required for transaction's authorization in general. There must additional, one-off signatures corresponding to every extra signer in this list, even if the signature is not otherwise required by the sourceAccount or operations.
+
+The additional signers can be of any type besides a pre-authorized transaction signer, since to pre-authorize a transaction, you need to know its hash, but the hash must include the extra signers. This Catch-22 relationship means including this type of extra signer will return a [`MALFORMED` error](#result-codes).
```
Comment (sisuresh) [inline content/docs/glossary/transactions.mdx:75]: Returning `MALFORMED` when trying to use pre-auth signers in extra signers didn't make it into the CAP. I discussed this with Nico and we landed on not special casing this.

Comment (Shaptic) [reply content/docs/glossary/transactions.mdx:75]: Is there no error, then? Just a logic error on behalf of the submitter?

Comment (sisuresh) [reply content/docs/glossary/transactions.mdx:75]: It would end up failing with `txBAD_AUTH` because the signatures won't be satisfied.

Comment (Shaptic) [reply content/docs/glossary/transactions.mdx:75]: Ah yes

File: content/docs/glossary/transactions.mdx (hunk 20 above)
Comment (rice2000) [reply content/docs/glossary/transactions.mdx:58]: +1 on the suggestion.  Not only is it better for cut-and-paste, but it will also make the copy parallel the sections below, which start with "when the sequence age precondition is set..." and "when the ledger gap precondition is set..."

I think the titles serve a separate purpose: they're signposts, and they show up on the nav menu so people can find and jump to a given section.

**Review by rice2000 — APPROVED** [review]
Agreed with one existing suggestion, but other than that, these changes are great.  Clear, concise, and clean!

**Comment by Shaptic** [comment]
This is to be merged after testnet goes live, right? @rice2000

**Comment by rice2000** [comment]
> This is to be merged after testnet goes live, right? @rice2000

Yeah: let's merge it Monday.
